### ISSUE TITLE
Opening Vector ASC files

### ISSUE THREAD

**Issue by damienmaguire** [collin80/SavvyCAN#528]
So I have some can logs in vector asc format but Savvycan seems to not like displaying the data properly. I'm probably doing something stupid. Here is an example from the file :

date Wed Oct 5 07:57:38.037 am 2022
base hex  timestamps absolute
internal events logged
// version 9.0.0
Begin Triggerblock Wed Oct 5 07:57:38.037 am 2022
   0.000000 Start of measurement
   0.001132 CANFD   1 Rx        14e  A0058_Unknown                    0 0 8  8 00 00 00 00 00 00 00 00   242136  125   200000      dbc 46500250 460a0250 20011736 20010205
   0.001371 CANFD   1 Rx        171  A0066_Unknown                    0 0 8  8 00 00 07 76 23 00 00 00   230199  119   220000     62e7 46500250 460a0250 20011736 20010205
   0.001611 CANFD   1 Rx        2dd  A0094_Unknown                    0 0 8  8 ea 0c ff 40 0f fc 00 00   232031  120   220000     5846 46500250 460a0250 20011736 20010205

**Comment by collin80** [comment]
No, I'm sure it's my fault. Vector seems to have made a lot of different versions of their ASC file format and so far I've only seen about 3-4 different types. It isn't that uncommon that someone comes up with a file of a version I have not seen yet. I'll see what I can do to support this version as well.
